benches/justfile

```diff
@@ -9,8 +9,10 @@ commit := `git rev-parse HEAD`
 # Run CPU benchmarks
 bench +benches:
   #!/bin/sh
+  [ $(uname -m) = "x86_64" ] && FEATURES="asm" || FEATURES="default"
+
   for bench in {{benches}}; do
-    cargo criterion --bench $bench
+    cargo criterion --bench $bench --features $FEATURES
```

Comment:
```suggestion
    cargo criterion --bench $bench --features $FEATURES --message-format=json > "$bench-{{commit}}".json
```

Reply:
Note: this is the normal, original bench job. Not the CI one. Similarly the gpu-bench job does not write to json either.